src/index.js

```diff
@@ -271,7 +271,7 @@ export default class RNPickerSelect extends PureComponent {
 
     renderAndroidHeadless() {
         return (
-            <View style={{ borderWidth: 0 }}>
+            <View style={[{ borderWidth: 0 }, this.props.style.inputAndroidContainer]}>
```

Comment:
call this `headlessAndroidContainer` instead

Reply:
👍